GETTING_STARTED.txt

```diff
@@ -0,0 +1,20 @@
+Getting Started
+===============
+
+Once you are logged in you will be taken to the admin page. /admin/
+http://localhost:8000/admin/
+
+Please find the Importer link on the admin page.
+
+On the importer page, click Browse button and select the SAMPLE_REGISTRY.yaml file included in the RDRF repo.
+
+This will create a sample registry with a few simple elements, which will help you get started.
+
+Next steps:
+1. From the main admin page, add a WorkingGroup (This will be needed while adding a patient)
+2. From the header navigation menu, select Patient List and add a patient.
+3. You can use Resgistry Common Data Element, Registry Section, Registry Form to create your own data collection pages.
+4. You can use Survey Question, Surveys to create surveys for data collection.
```

Comment:
Unless we add instructions for setting up a PROMS site not sure we should mention this - better to just have a non-CIC like registry sample - I can add proms system documentation later

Reply:
removed Survey related step